rasa/nlu/featurizers/__init__.py

```diff
@@ -5,8 +5,10 @@
 
 class Featurizer(Component):
     @staticmethod
-    def _combine_with_existing_text_features(message, additional_features):
-        if message.get("text_features") is not None:
-            return np.hstack((message.get("text_features"), additional_features))
+    def _combine_with_existing_features(message, additional_features, attribute="text"):
+        if message.get("{0}_features".format(attribute)) is not None:
```

Comment:
Can you use `feature_name = MESSAGE_VECTOR_FEATURE_NAMES[attribute]` and then reuse ´feature_name` instead of `"{0}_features".format(attribute)`.

Reply:
Yes. Good point.